src/ProfileView.jl

```diff
@@ -568,6 +568,10 @@ function __init__()
             printstyled(io, "\n`using Cthulhu` is required for `$(exc.f)`"; color=:yellow)
         end
     end
+    if !Gtk4.initialized[]
+        error("Gtk4 not initialized")
```

Comment:
Is this supposed to be an `@error` or is the `return` afterwards surplus?

Reply:
You're right, the `return` is not needed -- I just removed it.